### PR TITLE
Converted all Ava and Sinon testing to Jest

### PR DESCRIPTION
**Pull request by mushfichowdhury-magic**
### 📦 Pull Request

Converting all Ava and Sinon code to Jest.

### ✅ Fixed Issues

- [List any fixed issues here like: Fixes #XXXX]

### 🚨 Test instructions

Everything should run with Jest on yarn test

### ⚠️ Don't forget to add a [semver](https://semver.org/) label!

- `patch`: Bug Fix?
- `minor`: New Feature?
- `major`: Breaking Change?
- `skip-release`: It's unnecessary to publish this change.

### CODE REVIEW

**Review by smithki — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
-export { MagicAdminSDK as Magic } from './core/sdk';
+export { MagicAdminSDK } from './core/sdk';
```
Comment (smithki) [inline src/index.ts:1]: Please revert this change! This would cause a breaking change to the public API!

**Review by smithki — DISMISSED** [review]

**Review by smithki — CHANGES_REQUESTED** [review]
See comment above

**Comment by mushfichowdhury-magic** [comment]
@smithki updated!

**Review by smithki — APPROVED** [review]

**Comment by smithki** [comment]
:rocket: PR was released in [`v1.4.0`](https://github.com/magiclabs/magic-admin-js/releases/tag/v1.4.0) :rocket:

**Pull request merged**